File: VB/Helpers/MyScrollBar.vb
Imports Microsoft.VisualBasic
Imports DevExpress.XtraEditors
Imports DevExpress.XtraEditors.ViewInfo
Imports System
Imports System.Drawing

Namespace WindowsApplication1
	<System.ComponentModel.DesignerCategory("")> _
	Public Class MyScrollBar
		Inherits VScrollBar
		Public Sub New()
		End Sub
		Protected Overrides Function CreateScrollBarViewInfo() As ScrollBarViewInfo
			Return New MyScrollBarViewInfo(Me)
		End Function
	End Class
End Namespace

File: CS/Helpers/MyScrollBar.cs
using DevExpress.XtraEditors;
using DevExpress.XtraEditors.ViewInfo;
using System;
using System.Drawing;

namespace WindowsApplication1
{
    [System.ComponentModel.DesignerCategory("")]
    public class MyScrollBar : VScrollBar
    {
        public MyScrollBar()
        {
        }
        protected override ScrollBarViewInfo CreateScrollBarViewInfo()
        {
            return new MyScrollBarViewInfo(this);
        }
    }
}